### ISSUE TITLE
Error: writing blob: adding layer with blob

### ISSUE THREAD

**Issue by hollysheep** [netbrain/zwift#172]
**Describe the bug**
After downloading 4,4GB the process ends with this error:

> Error: writing blob: adding layer with blob "sha256:831b8865d6969bf1e41db7c949b00182b646c3f9d15c6ee777b9aa683177a617": creating read-only layer with ID "d7fd64894da17b1a6f74653d70c1f45b70a075c4943b7783948421c0eb0355ff": Stat /home/$USER/.local/share/containers/storage/overlay/96740c9cdb7d0018ff3f5046cef323767ffe0b7aed9026914a5ed72bb4677aea/diff: no such file or directory
> Error: stat /home/$USER/.local/share/containers/storage/overlay/9098b99ecc512d0f2561360a7966b09ba7c1fdc95b693d983be9e58010251d07: no such file or directory
> Error - Error can't run zwift, check variables!

**To Reproduce**
1. installing podman 4.6.2 on Ubuntu 22.04
2. installing zwift using the install script
3. running 'zwift'
4. starts downloading all the stuff and ends with the error above. There is enough free space on the Disc.

**Expected behavior**
Running.

**Additional context**
Is it normal that when i run 'zwift' again ALL downloads start again? The file are still there why is all downloaded again?

**Comment by netbrain** [comment]
No, this is not normal. Sounds like you have an issue with one or more of the image layers. Maybe you ran with sudo privileges and now some files have wrong perms?

**Comment by hollysheep** [comment]
Works now on my old Notebook with Ubuntu 22.04 and Podman 4.3. I don't understand whats the problem on my new Notebook.

Is there a Way to copy&paste the downloaded files to an other machine running with the same Version of Ubuntu and Podman?

The folder /home/$USER/.local/share/containers/ has a size of 2,4GB but i downloaded more than 4,4GB? Is it enough to simply copy that folder to a new machine?

**Comment by netbrain** [comment]
Haven't had this issue before, so no clue. 

If I were you, I'd try to purge the faulty system of all images and containers and try downloading again. Maybe even reinstall podman, and making sure all files purged clean

**Comment by sHedC** [comment]
Well assuming your laptop is ok, have you looked at these two sounds like a podman issue.

https://github.com/containers/podman/issues/19377

https://stackoverflow.com/questions/72651641/podman-returns-error-processing-tar-fileexit-status-1-operation-not-supporte
